sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -318,14 +318,11 @@ Command *Scheduler::GraphBuilder::addHostAccessor(Requirement *Req) {
     printGraphAsDot("before_addHostAccessor");
   markModifiedIfWrite(Record, Req);
 
-  AllocaCommandBase *SrcAllocaCmd = nullptr;
-
-  if (Record->MAllocaCommands.empty())
-    SrcAllocaCmd = getOrCreateAllocaForReq(Record, Req, HostQueue);
-  else
-    SrcAllocaCmd = findAllocaForReq(Record, Req, Record->MCurContext);
+  AllocaCommandBase *SrcAllocaCmd =
```

Comment:
```suggestion
  AllocaCommandBase *HostAllocaCmd =
```
Actually it's not always source allocation command.